generators/app/templates/src/app/components/CustomTextInput/AnimatedCustomTextInput/styles.ts

```diff
@@ -0,0 +1,78 @@
+import { StyleSheet } from "react-native";
+import fonts from "@config/fonts";
+import { black, blue, transparent, gray, darkGray } from "@constants/colors";
+import { SIZES } from "@constants/fonts";
+
+const INPUT_PADDING = 5;
+const IMAGE_SIZE = 14;
+
+export default StyleSheet.create({
+  containerWithLabel: {
+    marginTop: 10,
+    alignSelf: "stretch",
+  },
+  inputContainer: {
+    flexDirection: "row",
+    height: 25,
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  label: {
+    ...fonts.baseFont,
+    backgroundColor: transparent,
+    fontSize: SIZES.XSMALL,
+    position: "absolute",
+    left: INPUT_PADDING,
+    marginTop: 3,
+  },
+  container: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  inputStyle: {
+    ...fonts.baseFont,
+    fontSize: SIZES.SMALL,
+    color: darkGray,
+    padding: 0,
+    marginHorizontal: 0,
+    marginBottom: 0,
+  },
+  singleInput: {
+    flex: 1,
+  },
+  bottomBorderGray: {
+    borderBottomWidth: 1,
+    borderBottomColor: black,
+  },
+  bottomBorderBlue: {
+    borderBottomWidth: 1,
+    borderBottomColor: blue,
+  },
+  bottomBorderRed: {
+    borderBottomWidth: 1,
+    borderBottomColor: "red",
+  },
+  bottomBorderLightGray: {
+    borderBottomWidth: 1,
+    borderBottomColor: gray,
+  },
```

Comment:
they all use the same width, maybe we can add this default style for all and just add the color later